gfx/common/x11_common.h

```diff
@@ -44,6 +44,10 @@ void x11_save_last_used_monitor(Window win);
 void x11_show_mouse(Display *dpy, Window win, bool state);
 void x11_windowed_fullscreen(Display *dpy, Window win);
 void x11_suspend_screensaver(Window win, bool enable);
+void x11_suspend_screensaver_xdg_screensaver(Window win, bool enable);
```

Comment:
> Should be static. If you want the dispatcher function above the implementations, prototypes work fine.

Happy to do so, although I don't really understand why since my C is rusty, and no other functions on this file are stataic.